recipes-omnect/initrdscripts/omnect-os-initramfs/common-sh

```diff
@@ -62,7 +63,9 @@ check_fs()
     # regular boot; fsck handles unclean state (i.e.; not unmounted)
     fsck -y /dev/omnect/${label} &> /tmp/fsck_out
     fsck_res=$?
+    fsck_content=$(</tmp/fsck_out)
     msg "fsck result for \"${label}\" is ${fsck_res}"
+    msg "${fsck_content}"
```

Comment:
why do you need an extra var?
`msg "$(</tmp/fsck_out)"` does not work?

Reply:
oh yes, will work also. changed.